src/api/resource-environment-variable.js

```diff
@@ -0,0 +1,13 @@
+export const makeUrl = (appName, envName, componentName, envVarName) => {
+  const encAppName = encodeURIComponent(appName);
+  const encEnvName = encodeURIComponent(envName);
+  const encComponentName = encodeURIComponent(componentName);
+  const encEnvVarName = encodeURIComponent(envVarName);
+  return `/applications/${encAppName}/environments/${encEnvName}/components/${encComponentName}/envvars/${envVarName}`;
```

Comment:
Use encoded **encEnvVarName** in path, not **envVarName**

Reply:
Is this file used at all? I can't find any imports.

Reply:
No, it is outdated (replaced with `resource-environment-variables.js`). Deleted now.